### PR TITLE
[LLM] IPEX auto importer

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

Auto import IPEX if XPU version is installed

### 1. Why the change?

Ensure IPEX is imported for XPU

### 3. Summary of the change 

* Add `ipex_importer.py` to bigdl.llm.utils.
* Add auto import to `bigdl.llm.__init.py`.

### 4. How to test?
- [ ] N/A
- [x] Unit test
- [x] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

**Review by MeouSker77 — APPROVED** [review]
LGTM

**Comment by qiyuangong** [comment]
@jason-dai any comment?

**Comment by jason-dai** [comment]
Add an ENV to control the import

File: python/llm/src/bigdl/llm/__init__.py [hunk 1]
```diff
@@ -21,7 +21,11 @@
 
 from .convert_model import llm_convert
 from .optimize import optimize_model
+import os
 
-# Import Intel Extension for PyTorch as ipex if XPU version is installed
-from .utils.ipex_importer import ipex_importer
-ipex_importer.import_ipex()
+# Default is true, set to false to avoid auto importing Intel Extension for PyTorch.
+AUTO_IMPORT_IPEX = os.getenv("AUTO_IMPORT_IPEX", 'True').lower() in ('true', '1', 't')
```
Comment (jason-dai) [inline python/llm/src/bigdl/llm/__init__.py:27]: 1. Chane to `BIGDL_IMPORT_IPEX`
2. Set default to false and do more testing after merge

Comment (qiyuangong) [reply python/llm/src/bigdl/llm/__init__.py]: Done

**Pull request merged**